docs/quickstart/set-up-a-connection.md

```diff
@@ -1,44 +1,55 @@
 
-![](../.gitbook/assets/getting-started-connection.png)
+## Configure the connection
 
-## Check the logs of your first sync
+Once you've chosen your source and destination, you'll be able to configure the connection. You can refer to [this page](https://docs.airbyte.com/cloud/managing-airbyte-cloud/configuring-connections) for more information on each available configuration. For this demo, we'll simply set the **Replication frequency** to a 24 hour interval and leave the other fields at their default values.
 
-After you've completed the onboarding, you will be redirected to the source list and will see the source you just added. Click on it to find more information about it. You will now see all the destinations connected to that source. Click on it and you will see the sync history.
+![Connection config](../.gitbook/assets/set-up-a-connection/getting-started-connection-config.png)
 
-From there, you can look at the logs, download them, force a sync and adjust the configuration of your connection.
+Next, you can toggle which streams you want to replicate, as well as setting up the desired sync mode for each stream. For more information on the nature of each sync mode supported by Airbyte, see [this page](https://docs.airbyte.com/understanding-airbyte/connections/#sync-modes).
 
-![](../.gitbook/assets/getting-started-logs.png)
+Our test data consists of a single stream cleverly named `Test Data`, which we've enabled and set to `Full Refresh - Overwrite` sync mode.
 
-## Check the data of your first sync
+![Stream config](../.gitbook/assets/set-up-a-connection/getting-started-connection-streams.png)
 
-Now let's verify that this worked:
+Click **Set up connection** to complete your first connection. Your first sync is about to begin!
 
-```bash
-cat /tmp/airbyte_local/json_data/_airbyte_raw_pokemon.jsonl
-```
+## Connector Dashboard
+
+Once you've finished setting up the connection, you will be automatically redirected to a dashboard containing all the tools you need to keep track of your connection.
+
+![Connection dashboard](../.gitbook/assets/set-up-a-connection/getting-started-connection-success.png)
+
+Here's a quick breakdown of each tab:
+
+1. The **Status** tab shows you an overview of your connector's ongoing health.
+2. The **Job History** tab allows you to check the logs for each sync. If you encounter any errors or unexpected behaviors during a sync, checking the logs is always a good first step to finding the cause and solution.
+3. The **Replication** tab allows you to modify the configurations you chose during the connection setup.
+4. The **Transformation** tab allows you to configure [normalization and transformation settings](https://docs.airbyte.com/understanding-airbyte/basic-normalization) if supported by the connector.
```

Comment:
We actually went away with the Transformation tab with Destinations V2 - I think we can skip this setp